CMakeLists.txt

```diff
@@ -132,7 +134,7 @@ else()
 	if(LIBDRM_FOUND)
 		add_compile_definitions(SUNSHINE_BUILD_DRM)
 		include_directories(${LIBDRM_INCLUDE_DIRS})
-		list(APPEND PLATFORM_LIBRARIES ${LIBDRM_LIBRARIES})
+		list(APPEND PLATFORM_LIBRARIES ${LIBDRM_LIBRARIES} PkgConfig::LIBCAP)
```

Comment:
With find_package(), you could replace `PkgConfig::LIBCAP` with ${LIBCAP_LIBRARIES}